src/sass/mixins/_Icon.Mixins.scss

```diff
@@ -42,7 +42,7 @@
     height: 100%;
     margin:  0;
     padding: 0;
-    vertical-align: top;
+    // vertical-align: top;
```

Comment:
And with this, was there a CSS browser compat reason for removing this?

Reply:
Since this rule has a display:block declaration, vertical align wont be applied.